crates/sozo/src/commands/options/account.rs

```diff
@@ -64,6 +63,11 @@ impl AccountOptions {
     }
 
     fn signer(&self, env_metadata: Option<&Environment>) -> Result<LocalWallet> {
+        // INVARIANT: We assume that from commandline we can either specify `private_key` along with
+        // `account_address` or `keystore_path` along with `keystore_password`.
```

Comment:
I would rephrase this comment, as `account_address` is always needed.
Then we may have `private_key`, or `keystore_*` combo.